packages/venia-ui/__mocks__/react-intl.js

```diff
@@ -7,6 +7,6 @@ const intl = reactIntl.createIntl({
 
 module.exports = {
     ...reactIntl,
-    FormattedMessage: jest.fn(({ defaultMessage }) => defaultMessage),
+    FormattedMessage: jest.fn(({ defaultMessage, id }) => defaultMessage || id),
```

Comment:
This should be discussed; @revanth0212 solved this by providing his own mock in each test file, which I think we should replacing this one for all tests. Added this simple solution if we're okay just swapping components for one of these props (fallback to id when defaultMessage isn't provided).

Reply:
```jsx
jest.mock('react-intl', () => ({
    FormattedMessage: props => (
        <div componentName="Formatted Message Component" {...props} />
    )
}));
```
I usually do it like this because while adding snapshot tests it is important to capture all the props that a component is provided.

If we did this instead,
```jsx
FormattedMessage: jest.fn(({ defaultMessage, id }) => defaultMessage || id)
```
There can be false positives because this component returns the same even if the value of `defaultMessage` changes from `''` to `null` or `false` for instance.

Reply:
Checkout https://github.com/magento/pwa-studio/pull/2792 

I have updated the mock. The snapshots now can store more information about the formatted message component including child components if any.